middle_end/flambda2/terms/flambda.ml

```diff
@@ -896,7 +896,7 @@ and print_let_symbol ppf t =
   in
   let flattened, body = flatten_let_symbol t in
   match flattened with
-  | [] -> assert false
+  | [] -> fprintf ppf "@[<v>let_symbol <empty> in@ %a@]" print body
```

Comment:
Do you actually need this patch ? I fixed similar errors in #428 by printing deleted code.

Reply:
You're right, this is not needed, i'll remove that.